app/src/main/java/io/gnosis/safe/ui/safe/balances/SafeBalancesFragment.kt

```diff
@@ -27,9 +35,24 @@ class SafeBalancesFragment : SafeOverviewBaseFragment<FragmentSafeBalancesBindin
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            val adapter = BalancesPagerAdapter(this@SafeBalancesFragment)
+            balancesContent.adapter = adapter
+            TabLayoutMediator(balancesTabBar, balancesContent, true) { tab, position ->
+                if (position % 2 == 0) {
```

Comment:
why are you not using your newly created Tabs.COINS and Tabs.COLLECTIBLES here instead of having some weird positional logic ;)

Reply:
oops forgot that part xD it felt weird that I could simply make that private xD